azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -66,6 +66,8 @@
       "azkaban.executorinfo.refresh.maxThreads";
   private static final String AZKABAN_MAX_DISPATCHING_ERRORS_PERMITTED =
     "azkaban.maxDispatchingErrors";
+  private static final String AZKABAN_WEBSERVER_CLEANER_WAIT_INTERVAL =
+          "azkaban.webserver.cleaner.waitInterval.ms";
```

Comment:
azkaban is following this kind of notion: azkaban.activeexecutor.refresh.milisecinterval. we can follow similar: azkaban.webserver.cleaner.wait.milisecinterval

Reply:
changed